### PR TITLE
ref task #4818 Init project

### PR DESCRIPTION
**Pull request by thanh01684265170**
feature: Init project
Link: https://edu-redmine.framgia.vn/issues/4818

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -63,3 +63,4 @@ fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
 fastlane/readme.md
+.idea/
```
Comment (DoanVanToan) [inline .gitignore]: add end of file.

File: app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "hvcnbcvt_uddd.toeic600"
```
Comment (DoanVanToan) [inline app/build.gradle]: change app id and package name to com.framgia.toeic

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            android:theme="@style/AppTheme.NoActionBar">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
+
+</manifest>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: add end of file

File: app/src/main/java/com/framgia/toeic/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    @Override
+    public void onBackPressed() {
+        DrawerLayout drawer = (DrawerLayout) findViewById(R.id.drawer_layout);
+        if (drawer.isDrawerOpen(GravityCompat.START)) {
+            drawer.closeDrawer(GravityCompat.START);
+        } else {
+            super.onBackPressed();
+        }
+    }
+
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/toeic/MainActivity.java]: remove bl

File: app/src/main/res/layout/content_main.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:layout_behavior="@string/appbar_scrolling_view_behavior"
+    tools:context=".MainActivity"
+    tools:showIn="@layout/app_bar_main">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/content_main.xml]: remove hardcode

**Comment by thanh01684265170** [comment]
ready

**Pull request merged**